mandatory/2-function-creation.js

```diff
@@ -15,15 +20,23 @@ Complete the function to check if the variable `num` satisfies the following req
 Tip: use logical operators
 */
 
-function validate(num) {}
-
+function validate(num) {
+  if (typeof num === "number" && num % 2 === 0 && num <= 100) {
+    return true;
+  } else return false;
+}
 /* 
 Write a function that returns a copy of the given array arr, but with the element at the given index, index removed.
 The function must NOT change the original array, arr.
 */
 
 function remove(arr, index) {
-  return; // complete this statement
+  let newArr = [];
+  for (elem of arr) {
+    newArr.push(elem);
+  }
```

Comment:
Well done :+1: 
Is there a way to return a copy of an array without having to iterate through each element? ;)